src/labels_by_user_input.py

```diff
@@ -10,25 +12,60 @@
 )
 
 
+LABELS_SET = {
+    "hold": "do-not-merge/on-hold",
+    "wip": "do-not-review/wip",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+# Newline characters aren't supported in f-strings
+UNSUPPORTED_LABELS = """
+You're trying to add/remove an unsupported label.
+Supported labels -
+{}
+To remove a label - /un<LABEL_NAME>
+""".format(
+    "\n".join([f"/{key} to set {LABELS_SET[key]}" for key in LABELS_SET])
+)
+
+
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    last_commit = list(pull.get_commits())[-1]
-    if f"/{LABEL_VERIFIED}".lower() in body and LABEL_VERIFIED not in get_labels(
-        pull=pull
-    ):
-        add_label(pull=pull, label=LABEL_VERIFIED)
-        set_commit_status_success_verify(commit=last_commit)
-
-    if f"/un{LABEL_VERIFIED}".lower() in body:
-        remove_label(pull=pull, label=LABEL_VERIFIED)
-        set_commit_status_pending_no_verify(commit=last_commit)
-
-    if f"/{LABEL_APPROVE}".lower() in body and LABEL_APPROVE not in get_labels(
-        pull=pull
-    ):
-        add_label(pull=pull, label=LABEL_APPROVE)
-        set_commit_status_success_approve(commit=last_commit)
-
-    if f"/un{LABEL_APPROVE}".lower() in body:
-        remove_label(pull=pull, label=LABEL_APPROVE)
-        set_commit_status_pending_no_approve(commit=last_commit)
+    user = data["comment"]["user"]["login"]
+    pr_labels = get_labels(pull=pull)
+    comment_labels = re.findall("(?:(?<=\\s)|(?<=^))/\\S*", body)
+
+    for label in comment_labels:
+        unlabel = label.startswith("/un")
+        # If user label doesn't start with '/un', only '/' is stripped
+        stripped_label = label.lower().lstrip("/un")
+
+        if stripped_label not in LABELS_SET:
+
+            print(UNSUPPORTED_LABELS)
+            return pull.create_comment(body=f"Hey @{user},{UNSUPPORTED_LABELS}")
+
+        target_label = LABELS_SET[stripped_label]
+        verified = LABEL_VERIFIED in target_label
+        approved = LABEL_APPROVE in target_label
+        last_commit = list(pull.get_commits())[-1]
+
+        if target_label in pr_labels and unlabel:
+            remove_label(pull=pull, label=target_label)
```

Comment:
Please add a print about removing label.

Reply:
ACK

Reply:
There is a fix for approve comment
https://github.com/myakove/set-labels/pull/31

I made a PR so you can see whet is needed, i'm not merging it to avoid conflicts.
Please use this logic in this PR.

Reply:
Done

Reply:
remove and add approve label should be done only if approver

Reply:
@myakove lines 52-54 verify that

Reply:
line 59 remove the label regardless the approver.

Reply:
and maybe do f"{approver}/{target_label}" for the label?
@rnetser WDYT?

Reply:
yes; can be useful